content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md

```diff
@@ -214,22 +254,14 @@ Here are values used today for each Windows Server version.
 | Windows Server version 1809          | 10.0.17763             |
 | Windows Server version 1903          | 10.0.18362             |
 
-
-### Simplifying with RuntimeClass
-
-[RuntimeClass] can be used to simplify the process of using taints and tolerations. 
-A cluster administrator can create a `RuntimeClass` object which is used to encapsulate these taints and tolerations.
-
-
-1. Save this file to `runtimeClasses.yml`. It includes the appropriate `nodeSelector` 
-for the Windows OS, architecture, and version.
+A sample runtimeclass with nodeSelectors to include build numbers is shown below
 
 ```yaml
 apiVersion: node.k8s.io/v1
 kind: RuntimeClass
 metadata:
   name: windows-2019
-handler: 'docker'
+handler: 'runhcs-wcow-process'
```

Comment:
Can we use tabs to show handles for docker and the containerd default handler similar to OS tabs at https://kubernetes.io/docs/setup/production-environment/container-runtimes/#cri-o?

A lot of windows users are still running with docker (unfortunately) and this can be a pretty big change for them.

Reply:
I need some help from @sftim or someone from familiar with that `<tab>` tag. I am not sure if I am having correct tags.